spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Message.java

```diff
@@ -13,17 +13,23 @@
  * @see <a href="https://docs.oasis-open.org/sarif/sarif/v2.1.0/os/sarif-v2.1.0-os.html#_Toc34317459">3.11 message object</a>
  */
 final class Message {
+    @NonNull
+    final String text;
+
     @NonNull
     final List<String> arguments;
 
-    Message(@NonNull List<String> arguments) {
+    Message(@NonNull String text, @NonNull List<String> arguments) {
```

Comment:
This will affect any other code that might use this class.

Reply:
I removed the parameter from the constructor. Accessing the field is the only way to set the value.

Some null protection was added and the code calling it was refactored.